apps/frontend/src/components/cards/controltable/ControlRowHeader.vue

```diff
@@ -263,5 +268,22 @@ export default class ControlRowHeader extends mixins(HtmlSanitizeMixin) {
       return {label: cci, url: '', description: this.descriptionForTag(cci)};
     });
   }
+  showLegacy(control: any) {
```

Comment:
@HenryXiaoHX for future reference, we want to avoid using 'any' as much as possible since it essentially disables the type system and lets anything pass.  Types are great in how much safer your code ends up being, but actually finding/writing down the correct type can be frustrating.  Using 'any' as an escape should only be done in the rarest of circumstances since it weakens the typing that you've applied in all of the rest of the application.